client/components/button/style.scss

```diff
@@ -219,7 +219,7 @@ button {
 .button.is-borderless {
 	border: none;
 	background: none;
-	color: var( --color-neutral-700 );
+	color: var( --color-neutral-500 );
```

Comment:
This is equal to `--color-text-subtle` I believe. Do you mind using that instead since this is the color of the text?

Reply:
They are, started something at #30328 for that. :)